mz_crypt_apple.c

```diff
@@ -25,6 +26,7 @@ enum {
 
 CCCryptorStatus CCCryptorGCMReset(CCCryptorRef cryptorRef);
 CCCryptorStatus CCCryptorGCMAddIV(CCCryptorRef cryptorRef, const void *iv, size_t ivLen);
+CCCryptorStatus CCCryptorGCMaddAAD(CCCryptorRef cryptorRef, const void *aData, size_t aDataLen);
```

Comment:
Interesting, looks like this wrong cased name. But macOS docs, show
<img width="272" alt="image" src="https://user-images.githubusercontent.com/1364432/236474680-54abd9fa-af11-4c07-b6b5-cd3ea3d2f18d.png">
But odd that they both show the same macOS and iPhone minimum versions. Perhaps it was an internal name?

Reply:
Let's change to correct case since it is the same `AVAILABLE_STARTING` version for both functions.